redaxo/src/addons/cronjob/install.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+rex_sql::factory()->setQuery('
+    CREATE TABLE IF NOT EXISTS `'.rex::getTable('cronjob').'` (
```

Comment:
können wir das irgendwie "schöner" realisieren?

die definition in diesem schema hier oben ist ja "egal" oder?, unten durch die ensureColumns() wird ja sowieso erst das "finale" schema erreicht?

vllt stattdessn sowas wie `rex_sql_table::new(rex::getTable('cronjob'), $tableExtra)` oder `rex_sql_table::create(rex::getTable('cronjob'), $tableExtra)` oder sowas

$tableExtra wäre dann dinge wie ENGINE oder CHARSET, könnte aber auch optional sein diese 2 settings im konrekten fall ja meistens gleich sind